src/translib/authorize.go

```diff
@@ -40,30 +40,41 @@ func init() {
 }
 
 //TODO:Subscribe to UserDB changes and repopulate the cache on any notification
-func isUserAuthorizedForSet(user string) bool {
-	//TODO:Need code to get role from username and then authorize based on that
-	//This is a temporary code, this needs to be fixed
+// func isUserAuthorizedForSet(user string) bool {
+// 	//TODO:Need code to get role from username and then authorize based on that
+// 	//This is a temporary code, this needs to be fixed
 
-	if ((user == "") || (user == "admin")) {
-		return true
-	}
+// 	if ((user == "") || (user == "admin")) {
+// 		return true
+// 	}
 
-	return false
-}
+// 	return false
+// }
 func isAuthorizedForSet(req SetRequest) bool {
 	// TODO: Allow empty user for now
```

Comment:
Can delete this comment if we're getting rid of the "empty user" allowance

Reply:
Deleted